planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -573,6 +576,11 @@ public Planetiler fetchWikidataNameTranslations(Path defaultWikidataCache) {
         fetchWikidata);
     useWikidata = fetchWikidata || arguments.getBoolean("use_wikidata", "use wikidata translations", true);
     wikidataNamesFile = arguments.file("wikidata_cache", "wikidata cache file", defaultWikidataCache);
+    wikidataMaxAge =
+      arguments.getDuration("wikidata_max_age", "Maximum age of Wikidata translations (0 = disabled)", "720H");
+    // limit of 100_000 is roughly 5% of total amount of entries for whole Planet
+    wikidataUpdateLimit = arguments.getInteger("wikidata_update_limit",
+      "Limit on how many old translations to update during one download (0 = disabled)", 100_000);
```

Comment:
I think the default behavior should minimize load on wikidata infra, this seems like more of an advanced feature we'd want people to opt into. Can we make the default max age be 0?

Reply:
Makes sense, adjusted.